src/Makefile

```
@@ -1,13 +1,10 @@
-CXXFLAGS=-g -O3 -Wall -Werror -Wno-error=unknown-pragmas -std=c++11 -pthread -I./include -I./vendor -fopenmp
-LDLIBS=-lz
+CXXFLAGS=-g -O2 -Wall -Werror -Wno-error=unknown-pragmas -std=c++11 -pthread -I./include -I./vendor -fopenmp
+LDLIBS=-lz 
```

Please revert the changes to this chunk.

---

Please remove the stray white space here.
```suggestion
LDLIBS=-lz
```